### PR TITLE
Fixes #7081: Improve rudder server requirements formatting

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/7081

### CODE REVIEW

File: 00_introduction/21_architecture_root_server.txt [hunk 1]
```diff
@@ -2,58 +2,45 @@
 
 === Specifications for Rudder Root Server
 
-==== Hardware specifications
+==== Hardware specifications and sizing
 
-A dedicated server is strongly recommended to deploy Rudder Root Server.
+A dedicated server is strongly recommended, either physical or virtual with at least one dedicated core.
+Rudder Server is running on both 32 (if available) and 64 bit versions of every supported Operating System.
```
Comment (jooooooon) [inline 00_introduction/21_architecture_root_server.txt]: s/is running/runs/

Comment (jooooooon) [inline 00_introduction/21_architecture_root_server.txt]: Ah, I see now this was already in the previous version. If you can change it, cool, if not don't worry, it can go in another PR.

File: 00_introduction/21_architecture_root_server.txt [hunk 2]
```diff
@@ -2,58 +2,45 @@
...
-The base requirement is to have a server with 2GB of RAM. 
-On production environment, when managing more than 50 nodes, it is recommended to 
-dedicate at least 4GB of RAM to the server.
+When managing more than 1000 nodes, we also advice you to use a multiserver
+installation for Rudder as described in chapter <<multiserver-rudder, Multiserver Rudder>>.
 
-Then, the default rule is to add 1GB of RAM by 500 nodes above 1000 nodes,
-so for example for 1500-2000 nodes, you need at minimum 6GB of RAM, and
-for 2500-3000 you need 8GB.
+For large installation, you should also tune the amount of RAM given to:
```
Comment (jooooooon) [inline 00_introduction/21_architecture_root_server.txt]: large installation**s**

File: 00_introduction/21_architecture_root_server.txt [hunk 3]
```diff
@@ -2,58 +2,45 @@
...
-+/opt/rudder/etc/rudder-web.properties+ file with the options:
- 
-+rudder.batch.reportscleaner.archive.TTL=30+
-
-+rudder.batch.reportscleaner.delete.TTL=90+
-
 For example, an installation managing 150 nodes, with an average of 
 100 Directives by node, and keeping execution reports for one week 
-should anticipate the use of between 20Go and 60Go of disk size for PostgreSQL. 
+should anticipate the use of between 20Go and 60Go of disk size for PostgreSQL.
```
Comment (jooooooon) [inline 00_introduction/21_architecture_root_server.txt]: s/Go/GB/g

File: 00_introduction/21_architecture_root_server.txt [hunk 4]
```diff
@@ -2,58 +2,45 @@
...
+* less than 50 nodes: 2GB
+* between 50 and 1000 nodes: 4GB
+* more than 1000 nodes: 4GB + 1GB of RAM by 500 nodes above 1000.
 
-The amount of RAM needed by the server depends mainly on the number of managed nodes.
-
-The base requirement is to have a server with 2GB of RAM. 
-On production environment, when managing more than 50 nodes, it is recommended to 
-dedicate at least 4GB of RAM to the server.
+When managing more than 1000 nodes, we also advice you to use a multiserver
```
Comment (jooooooon) [inline 00_introduction/21_architecture_root_server.txt]: I see you changed "advise" to "advice" here. This is actually incorrect (advice is the noun, and advise is the verb, so "advise" is right here). However, it may be better to replace it with "recommend".

**Comment by jooooooon** [comment]
Aside from the small details I mentioned above, this looks like a very welcome change.

However, why include it only in master? This is just clarifying and formatting (the bug fixing of documentation) and applies to all existing versions of Rudder (as far as I can tell). I suggest targeting our current ESR version instead, to make sure any new users starting out get this info without waiting for master to be released as a version.

**Comment by amousset** [comment]
Retaregted in #128.
